html/controls/operator.js

```diff
@@ -1235,92 +1201,92 @@ function createTimeoutDialog() {
 		firstJamListed[p] = newFirst;
 		lastJamListed[p] = newLast;
 	}
-	
+
 	function processTimeout(k, v) {
 		var id = k.Timeout;
 		if (id == 'noTimeout') { return; }
 		var p = Number(k.Period);
 		var prefix = 'ScoreBoard.Period('+k.Period+').Timeout('+id+')';
-		var row = table.find('tr.Timeout[toId='+id+']');
-		if (k.field == 'Id' && v == null && row.length > 0) {
-			row.remove();
-			return;
-		}
-		if (k.field == 'PrecedingJamNumber') {
-			row.remove();
-			row = [];
-		}
-		if (v != null && row.length == 0) {
-			var jam = Number(WS.state[prefix+'.PrecedingJamNumber']);
-			var dur = isTrue(WS.state[prefix+'.Running']) ? 'Running' : _timeConversions.msToMinSec(WS.state[prefix+'.Duration']);
-			var pc = _timeConversions.msToMinSec(isTrue(WS.state[prefix+'.Running']) ?
-					WS.state['ScoreBoard.Clock(Period).Time'] :
-					WS.state[prefix+'.PeriodClockEnd']);
-			var type = WS.state[prefix+'.Owner'] + '.' + WS.state[prefix+'.Review'];
-			var review = isTrue(WS.state[prefix+'.Review']);
-			var retained = isTrue(WS.state[prefix+'.RetainedReview']);
-			row = $("<tr>").addClass("Timeout").attr("toId", id).attr('period', k.Period).attr('jam', jam)
-				.append($('<td>').addClass('Period').append(periodDropdownTemplate.clone().val(p).change(function() {
-					WS.Set(prefix+'.PrecedingJam', WS.state['ScoreBoard.Period('+$(this).val()+').CurrentJam']);
-				})))
-				.append($('<td>').addClass('Jam').append(jamDropdownTemplate[p].clone().val(jam).change(function() {
-					WS.Set(prefix+'.PrecedingJam', WS.state['ScoreBoard.Period('+p+').Jam('+$(this).val()+').Id']);
-				})))
-				.append($('<td>').addClass('Duration').text(dur))
-				.append($('<td>').addClass('PerClock').text(pc))
-				.append($('<td>').addClass('Type').append(typeDropdownTemplate.clone().val(type).change(function() {
-					var parts = $(this).val().split('.');
-					WS.Set(prefix+'.Owner', parts[0]);
-					WS.Set(prefix+'.Review', isTrue(parts[1]));
-				})))
-				.append($('<td>').addClass('Retained').append($('<button>').toggleClass('Hide', !review).toggleClass('Active', retained)
-						.text('Retained').button().click(function () {
-							WS.Set(prefix+'.RetainedReview', !isTrue(WS.state[prefix+'.RetainedReview']));
+				var row = table.find('tr.Timeout[toId='+id+']');
```

Comment:
Why does the indentation change here?

Reply:
I reindented the entire file, so anything that was inconsistent would also have changed.

Reply:
But this line should be at the same level as the line above.

Reply:
Sometimes it does this, I'm not sure why. Worked around it.